longest_palindrome.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ *                  Longest Palindrome
+ *
+ * Leetcode : https://leetcode.com/problems/longest-palindrome/ 
+ * 
+ * Time complexity : O(N)
+ * Space complexity : O(N) 
```

Comment:
Check space complexity. Since you just have lower case characters , so it will be O(26) will be O(1)